packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/queued_item_store/drift/drift_queued_item_store.g.dart

```diff
@@ -26,9 +26,10 @@ class $DriftQueuedItemsTable extends DriftQueuedItems
   @override
   List<GeneratedColumn> get $columns => [id, value];
   @override
-  String get aliasedName => _alias ?? 'drift_queued_items';
+  String get aliasedName => _alias ?? actualTableName;
   @override
-  String get actualTableName => 'drift_queued_items';
+  String get actualTableName => $name;
+  static const String $name = 'drift_queued_items';
```

Comment:
question: are these changes expected?

Reply:
I think so, it is a generated file. I think it was missing in #4029 and so was added by aft. #4029 has similar changes for `packages/storage/amplify_storage_s3_dart/lib/src/storage_s3_service/transfer/database/database_io.g.dart` and for some reason did not update the `packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/queued_item_store/drift/drift_queued_item_store.g.dart`

Reply:
It is showing up in the version bump commit (5600e2f2e48234b827ce96d1c5f4842701d79403) which seems odd.

shouldn't this have been in another commit?